internal/misc/http/helpers.go

```diff
@@ -45,3 +51,17 @@ func GetUserClaims(i kratos_client.Identity, cr hydra_client.OAuth2ConsentReques
 
 	return ret
 }
+
+func AddParamsToURL(u string, qs ...QueryParam) (string, error) {
```

Comment:
This was moved, but it still only used in the kratos module, not sure if we want to revert this change or not.

Also I think we should either use this function everywhere or throw it away (see https://github.com/canonical/identity-platform-login-ui/pull/398/files#diff-eee000f25821f185296fc9109030f323fca69b610a1b5ef787071dca62a8cf38R141, @shipperizer do you want to get rid of this function?)